apps/src/code-studio/showProjectAdmin.js

```diff
@@ -75,27 +75,41 @@ export default project => {
     });
   }
 
-  if (
-    $('.admin-abuse').length &&
-    (project.isProjectLevel() || !project.shouldHideShareAndRemix())
-  ) {
+  if ($('.admin-project-sharing').length) {
+    var sharingDisabled = project.getSharingDisabled();
+    var privateOrProfane = project.hasPrivacyProfanityViolation();
     var abuseScore = project.getAbuseScore();
-    if (abuseScore) {
-      $('.admin-abuse').show();
-      $('.admin-abuse-score').text(abuseScore);
-      $('.admin-abuse-reset').click(function() {
-        project.adminResetAbuseScore();
-      });
+    var abusive = project.exceedsAbuseThreshold();
+    if (sharingDisabled || privateOrProfane || abusive) {
+      $('.blocked').show();
+      $('.unblocked').hide();
+      if (sharingDisabled) {
+        $('.admin-sharing').show();
+      }
+      if (privateOrProfane) {
+        $('.privacy-profanity').show();
+      }
+      if (abusive) {
+        // The image moderation service sets the abuse score to 15 to
+        // differentiate from manual reports.
```

Comment:
tangential question, but if a featured project has an abusive image added to it the new starting score of -50, that won't be enough to "automatically" block the project, right? I think this is probably ok... because we choose who to feature, and the abusers are relatively unlikely to be selected... just confirming I understand the implications here.

Reply:
Yes, you are correct.  If an abusive image is added to a project while it is featured, then the increment on the abuse score via the image moderation service would not be enough to block the project.